backend/infrahub/workers/infrahub_async.py

```diff
@@ -204,3 +210,35 @@ async def _init_services(self, client: InfrahubClient) -> None:
         )
 
         self.service = service
+
+    async def set_git_global_config(self) -> None:
+        if INFRAHUB_PRODUCTION and not os.getenv("GIT_GLOBAL_CONFIG"):
+            os.makedirs(DEFAULT_GIT_GLOBAL_CONFIG_DIRECTORY, exist_ok=True)  # noqa: PTH103
+            os.environ["GIT_GLOBAL_CONFIG"] = DEFAULT_GIT_GLOBAL_CONFIG_FILE
+            self._logger.info(f"Set git config file to {DEFAULT_GIT_GLOBAL_CONFIG_FILE}")
+
+        if config.SETTINGS.git.user_name:
+            proc_name = await asyncio.create_subprocess_exec(
+                "git",
+                "config",
+                "--global",
+                "user.name",
+                config.SETTINGS.git.user_name,
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.PIPE,
+            )
+            await proc_name.wait()
+            self._logger.info("Git user name set")
+
+        if config.SETTINGS.git.user_email:
+            proc_email = await asyncio.create_subprocess_exec(
+                "git",
+                "config",
+                "--global",
+                "user.email",
+                config.SETTINGS.git.user_email,
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.PIPE,
+            )
+            await proc_email.wait()
```

Comment:
For these two commands we assume that everything will work fine. We should consider errors as well, what would it look like if any of these calls failed? Would the app crash or would the error be silently ignored?